### PR TITLE
Updated ironstrings to work with Python3.

### PR DESCRIPTION
**Pull request by unk0unk0unk0**
Modified some lines to work with Python3.
 - ironstrings.py: updated character handling.
 - strings.py: updated handling for return values of filter function.

### CODE REVIEW

File: python/flare/ironstrings/ironstrings.py [hunk 1]
```diff
@@ -159,14 +159,14 @@ def main():
             cnt_analyzed += 1
 
             # print stackstrings found in this function
-            f_ss = filter(lambda s: s.fva == fva, userData["stackstrings"])
+            f_ss = list(filter(lambda s: s.fva == fva, userData["stackstrings"]))
```
Comment (williballenthin) [inline python/flare/ironstrings/ironstrings.py:162]: good!

File: python/flare/ironstrings/strings.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 
 
 ASCII_BYTE = " !\"#\$%&\'\(\)\*\+,-\./0123456789:;<=>\?@ABCDEFGHIJKLMNOPQRSTUVWXYZ\[\]\^_`abcdefghijklmnopqrstuvwxyz\{\|\}\\\~\t"
-ASCII_RE_4 = re.compile("([%s]{%d,})" % (ASCII_BYTE, 4))
-UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE, 4))
+ASCII_RE_4 = re.compile(b"([%s]{%d,})" % (ASCII_BYTE.encode(), 4))
+UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE.encode(), 4))
```
Comment (williballenthin) [inline python/flare/ironstrings/strings.py:12]: recommend an explicit encoding here, probably `ascii`

Comment (mr-tz) [reply python/flare/ironstrings/strings.py:12]: yes, please use `r" !\"#\$...~\t".encode("ascii")`, then good to go, thanks!

**Comment by unk0unk0unk0** [comment]
Thanks.
I will commit fixed one based on your comments.

**Pull request closed without merging**